Scripts/travis.sh

```diff
@@ -37,26 +37,48 @@ execute_xcodebuild() {
     exit 1
   fi
 
-  retval_xcodebuild=0
+  retval_command=0
+  # Are we running a test?
+  [[ "${ACTION}" == *"test"* ]] && is_running_test=1 || is_running_test=0
```

Comment:
nit: this oneliner is neat! However for readability sake consider changing this to an if else statement.